database/migrations/20220531100155-update-sites.js

```diff
@@ -0,0 +1,195 @@
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    /**
+     * Add altering commands here.
+     *
+     * Example:
+     * await queryInterface.createTable('users', { id: Sequelize.INTEGER });
+     */
+    const transaction = await queryInterface.sequelize.transaction()
+    try {
+      await queryInterface.dropTable("site_members", { transaction })
+      await queryInterface.dropTable("sites", { transaction })
+
+      await queryInterface.createTable(
+        "sites",
+        {
+          id: {
+            allowNull: false,
+            autoIncrement: true,
+            primaryKey: true,
+            type: Sequelize.BIGINT,
+          },
+          repository_name: {
+            allowNull: false,
+            unique: true,
+            type: Sequelize.TEXT,
+          },
+          created_by: Sequelize.TEXT,
+          created_at: {
+            type: Sequelize.DATE,
+            allowNull: false,
+            defaultValue: Sequelize.fn("NOW"),
+          },
+          updated_at: {
+            type: Sequelize.DATE,
+            allowNull: false,
+            defaultValue: Sequelize.fn("NOW"),
+          },
+          agency: Sequelize.TEXT,
```

Comment:
I think it's a bad idea. It would require a database migration every time an agency is added. Also see above comment about error handling when a user is not in the database. Those problems would be magnified here, since this field is entered without any check at all.

Reply:
that's fair